webapp/src/components/root/root.jsx

```diff
@@ -98,7 +108,7 @@ export default class Root extends React.Component {
                             checked={this.state.attachToThread}
                             onChange={this.handleAttachChange}
                         />
-                        <b>{' Add to thread'}</b>
+                        <b>{isEditing ? 'Update thread' : ' Add to thread'}</b>
```

Comment:
If we are updating the todo, do we want to add it to a thread after the update? @aaronrothschild thoughts on this?

Reply:
Same will be for sending a todo.

Reply:
> If we are updating the todo, do we want to add it to a thread after the update? @aaronrothschild thoughts on this?

Yes, ideally the the user could provide a status update or reason for the change for example.